metabrainz/static/css/theme/boostrap/variables.less

```diff
@@ -414,7 +414,7 @@
 //##
 
 //=== Shared nav styles
-@nav-link-padding:                          10px 15px;
+@nav-link-padding:                          10px 7.5px;
```

Comment:
editing this file is unnecessary as well, simply the /theme/variables.less is enough as we use that custom file to generate  the .css :)
please revert these back too :)